.github/workflows/testing.yml

```diff
@@ -198,7 +198,7 @@ jobs:
         if: matrix.tarantool != 'brew' && matrix.tarantool != 'master'
 
       - name: Install tarantool build dependencies
-        run: brew install autoconf automake libtool openssl@1.1
+        run: brew install autoconf automake libtool openssl@1.1 git
         if: matrix.tarantool != 'brew' && steps.cache.outputs.cache-hit != 'true'
```

Comment:
Is it possible that we have no `git` installed on a GitHub Actions runner?

I created the following workflow in a testing repository:

```yml
name: Git on Mac OS

on: [push]

jobs:
  git_on_mac_os:
    strategy:
      matrix:
        runs-on:
          - macos-10.15
          - macos-11.0
    runs-on: ${{ matrix.runs-on }}
    steps:
      - run: git --version
```

And got the following:

* macos-10.15: git version 2.34.1.
* macos-11.0: git version 2.34.1.

Reply:
I really shouldn't install it in runner. That was rather a little insurance. Now fixed.